_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -233,6 +236,27 @@ $letter-spacing: (
 );
 
 // ----------------------------
+// #FORMS
+// ----------------------------
+
+// form inputs: heights
+// values are "rows" which maps to line-height
+// primarily used in textareas
+$input-height: (
+    short:          5,
+    base:           10,
+    tall:           15
+);
+
+// form inputs: widths
+$input-width: (
+    short:          150,
+    base:           300,
+    wide:           450,
+    block:          null
```

Comment:
This value cool and not going to cause us any trouble with your newly added function?